lint_models.yml

```diff
@@ -0,0 +1,16 @@
+name: Lint Models
+
+on: [pull_request]
+
+jobs:
+  lint-models:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: "actions/checkout@v2"
+      - uses: "actions/setup-python@v2"
+        with:
+            python-version: "3.8"
+      - name: Install SQLFluff
+        run: "pip install sqlfluff==0.3.6"
+      - name: Lint models
+        run: "sqlfluff lint models"
```

Comment:
I'd prefer if we:
* Did not assumes the SQL files are in `models/` (not all sqlfluff users use dbt)
* Only linted what changed

Something like:
```yaml
on:
  pull_request:

jobs:
  pre-commit:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v1
    - uses: actions/setup-python@v1
    - id: file_changes
      uses: trilom/file-changes-action@v1.2.4
      with:
        output: ' '
    - name: SQLFluff lint
      run: "sqlfluff lint ${{ steps.file_changes.outputs.files}}"
```

Excited to have a GitHub action :D

Reply:
Agreed -- we use an in-house tool rather than dbt, and typically our SQL files are in a `sql/` directory.

Reply:
Very happy to make it non-dbt-specific :)

I like having both versions available though (e.g. a "lint changed" and "lint all" version). Would you be open to me including two different hooks?

Also, I'm curious about naming this job `pre-commit` — I don't know much about job naming, is there a reason behind this decision!

Reply:
oops I'm only seeing this now! makes sense

The naming of `pre-commit` was just because I copied it from some other place, you can disregard :)